dartagnan/src/main/java/com/dat3m/dartagnan/program/memory/MemoryObject.java

```diff
@@ -92,29 +92,13 @@ public void setInitialValue(int offset, Expression value) {
         initialValues.put(offset, value);
     }
 
-    /**
-     * Updates the initial value at a certain field of this array.
-     *
-     * @param offset Non-negative number of fields before the target.
-     * @param value  New value to be read at the start of each execution.
-     */
-    public void appendInitialValue(int offset, Expression value) {
```

Comment:
Was this never used?

Reply:
No, that's why I could remove it :P. It was used when we still had the Boogie parser but became obsolete with the LLVM rework.